packages/streams/gpsd.js

```diff
@@ -61,23 +63,27 @@ function Gpsd(options) {
       },
     },
     parse: false,
+    reconnectThreshold: noDataReceivedTimeout,
+    reconnectInterval: noDataReceivedTimeout / 2
```

Comment:
This is used in `setInterval`. I don't think zero divided by zero is a good default, I'd guess that will result in a busy loop.

All existing users will use the default configuration when they get this update, so the default *must* be sane.

Reply:
So, sane is either have no disconnect, which is it now - or set it to some value; perhaps 30 seconds -- input is most wanted.

Reply:
30 seconds is probably as good as any reasonable value.

Reply:
I'll change the default to 30 seconds; but I guess the UI should reflect this - I'll figure out how to set the default there too.